glean/generated/gleansample.h

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Our team recommends not committing the generated code, when possible, as it hides some issues and might be problematic to debug if Glean changes and this is not changed: would you consider removing this?